infrastructure/transform/locals.tf

```diff
@@ -96,7 +96,8 @@ locals {
   msi_users = [
     { "name" : "${local.databricks_app_name}", "role" : "db_owner" },
     { "name" : "${azuread_group.ad_group_apps.display_name}", "role" : "db_datareader" },
+    local.data_scientists
```

Comment:
Don't think this should be in the msi_users? MSI is for system identities right? Why did you move this from the concat below?

Reply:
as below